wtransport/src/config.rs

```diff
@@ -236,7 +236,7 @@ impl ServerConfigBuilder<WantsTransportConfigServer> {
     /// Keep-alive packets prevent an inactive but otherwise healthy connection from timing out.
     ///
     /// `None` to disable, which is the default. Only one side of any given connection needs keep-alive
-    /// enabled for the connection to be preserved. Must be set lower than the idle_timeout of both
+    /// enabled for the connection to be preserved. Must be set lower than the `idle_timeout` of both
```

Comment:
Should this actually be `idle_timeout` or should it be `max_idle_timeout`? (And same below...)

Reply:
Yeah I guess `max_idle_timeout` makes more sense.

Probably ref to method to set, so it easier to navigate through the doc. That is, 
```
[`max_idle_timeout`](Self::max_idle_timeout)
```

what do you think?

Reply:
Agree, will do in a couple of hours when I'm back from badminton.

Reply:
Thanks

Reply:
Updated. Also added a thing to enable these warnings in the regular clippy runs.